odk1-src/form-question-types.rst

```diff
@@ -1043,8 +1043,20 @@ To include images as choices for select questions:
      make sure the media folder is placed in
      :file:`/sdcard/odk/forms/`.
 
-  
+.. _or-other:  
+
+Including "other" as an option
```

Comment:
s/option/choice.

It's not clear when you use option vs choice, but we should probably default to choice.